### PR TITLE
Added a Go Back Button on Reservation/New page

### PR DESCRIPTION
**Pull request by jchau905**
Added:
Go Back Button

image

### CODE REVIEW

File: app/views/reservations/new.html.erb [hunk 1]
```diff
@@ -4,6 +4,9 @@
   <%= simple_form_for([@game_world, @reservation]) do |f| %>
     <%= f.input :start_date, as: :date, html5: true %>
     <%= f.input :end_date, as: :date, html5: true %>
-    <%= f.submit %>
+    <%= f.submit%> <%= link_to "Go Back", game_world_path(@game_world), class: "btn btn-go-back" %>
   <% end %>
 </div>
+
+<div>
```
Comment (MikaelJG) [inline app/views/reservations/new.html.erb:11]: I'm not sure, is this an empty div?

File: app/assets/stylesheets/components/_button.scss [hunk 2]
```diff
@@ -50,3 +50,23 @@
...
+    cursor: default;
+    box-sizing: border-box;
+    background-color: buttonface;
+    color: buttontext;
+    padding: 1px 6px;
+    border-width: 0.5px;
+    border-style: outset;
+    border-color: grey;
+    border-image: initial;
+
```
Comment (MikaelJG) [inline app/assets/stylesheets/components/_button.scss:70]: I'm thinking these empty lines should be deleted.

**Review by MikaelJG — COMMENTED** [review]
Great Job Jamie, just a few things to verify. Thanks!

File: app/views/reservations/new.html.erb (hunk 1 above)
Comment (jchau905) [reply app/views/reservations/new.html.erb]: okay just pushed again and fixed the empty div

File: app/assets/stylesheets/components/_button.scss (hunk 2 above)
Comment (jchau905) [reply app/assets/stylesheets/components/_button.scss]: deleted those empty lines

**Pull request merged**